server/datastore/mysql/scheduled_queries.go

```diff
@@ -57,6 +57,7 @@ func (d *Datastore) insertScheduledQuery(tx *sqlx.Tx, sq *fleet.ScheduledQuery)
 	query := `
 		INSERT INTO scheduled_queries (
 			query_name,
+			query_id,
```

Comment:
I noticed that when we create a scheduled query, we don't save the query id, but when we modify it, we do. So I added this here for consistency.